lib/helpers/utils.js

```diff
@@ -155,5 +156,36 @@ exports.getLookupFieldsModelWithTypeOfSearch = async function getLookupFieldsMod
   return model;
 };
 
+exports.getLinkedObjectTypes = function getLinkedObjectTypes(meta) {
+  return {
+    ...meta.fields.filter(field => field.type === 'reference')
+      .reduce((obj, field) => {
+        if (!field.referenceTo.length) {
+          throw new Error(
+            `Empty referenceTo array for field of type 'reference' with name ${field.name} field=${JSON.stringify(
+              field, null, ' ',
+            )}`,
+          );
+        }
+        if (field.relationshipName !== null) {
+          // eslint-disable-next-line no-param-reassign
+          obj[field.relationshipName] = `${field.referenceTo.join(', ')} (${field.relationshipName})`;
+        }
+        return obj;
+      }, {}),
+    ...meta.childRelationships
```

Comment:
Better divide this structure to separate operations

Reply:
Was fixed there https://github.com/elasticio/salesforce-component/pull/166/files